view/template/mail/_settingsForm.php

```diff
@@ -7,12 +7,19 @@
   Javascript is required to securely send your unsubscribe information. Email <a href="mailto:help@lbry.io" class="link-primary">help@lbry.io</a> for manual unsubscription.
 </noscript>
 <form id="<?php echo $formId ?>" novalidate style="display: none">
-  <?php if ($error): ?>
-    <div class="notice notice-error spacer1"><?php echo $error ?></div>
-  <?php endif ?>
-  <div class="notice notice-success hide">Your email preferences have been updated.</div>
+  <div style="min-height: 48px">
+    <?php if ($error): ?>
+      <div class="notice notice-error spacer1"><?php echo $error ?></div>
+    <?php endif ?>
+    <div class="notice notice-success hide">Your email preferences have been updated.</div>
+  </div>
+
+
   <section class="email-section">
-    <h4>Where do you want to receive email?</h4>
+    <h4><?php echo count($emails ) > 1 ? 'Receiving Addresses' : 'Do You Want To Receive Mail?' ?></h4>
```

Comment:
I think part of the right answer to reducing confusion here is to change the behavior for 1 vs. many emails